lib/suspenders/generators/static_generator.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails/generators'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.